src/microbe_stage/systems/MicrobeReproductionSystem.cs

```diff
@@ -368,6 +368,13 @@ protected override void PostUpdate(float state)
             CalculateFreeCompoundsAndLimits(gameWorld!.WorldSettings, organelles.HexCount, false,
                 reproductionDelta);
 
+        if (entity.Has<MicrobeControl>())
```

Comment:
This system needs `[ReadsComponent(typeof(MicrobeControl))]` attribute.  This skip should also be earlier in the system, on line 227, I'd say.

Also unless I missed it, the multicell system also needs this change.